.azure-pipelines/windows-visual-studio.yml

```diff
@@ -0,0 +1,47 @@
+steps:
+  - script: |
+      :: Use Windows CRLF line endings for checked-out text files
+      git config --global core.autocrlf true
+      set
+    displayName: Print environment variables
+  - checkout: self
+    fetchDepth: 1
+  - script: |
+      bash --version
+      sh --login .azure-pipelines/windows-visual-studio.sh
+
+  - script: |
+      echo on
+      call "%VSINSTALLDIR%\VC\Auxiliary\Build\vcvarsall.bat" %ARCH%
+      set LDC_DIR=ldc2-%LDC_VERSION%-windows-multilib
+      set VISUALD_INSTALLER=VisualD-%VISUALD_VER%.exe
+      set PWD="%cd%"
```

Comment:
I suspect quotes in partial pathnames are no good, I wouldn't add them here, but when passing the argument "%PWD%\dmd2"